server/controllers/products.js

```diff
@@ -3,11 +3,33 @@
 const fetch = require('../lib/fetch/wrap')(require('node-fetch'));
 const ammit = require('../lib/ammit');
 const {json} = require('fetchres');
+const {itemTransform} = require('../lib/item-transform');
 
 const apiKey = process.env.BARRIER_GURU_API_KEY;
 
 const FIVE_YEARS = 5 * 365.25 * 24 * 60 * 60 * 1000;
 
+const getProducts = async (ammitParams, countryCode) => {
+	const { abVars, allocation } = await ammit(ammitParams);
```

Comment:
Minor thing, but spacing in this repo seems to be like this
```suggestion
	const {abVars, allocation} = await ammit(ammitParams);
```